.gitignore

```diff
@@ -2,4 +2,6 @@
 node_modules
 .secrets
 *.DS_STORE*
-.wayscript
\ No newline at end of file
+.wayscript
+.env
```

Comment:
Are we sure about ignoring this? We may want to add built-in initial `.env` values for the template

Reply:
I thought I would add a .env.example instead

Reply:
Lets stick with actual .env as there shouldnt be anything sensitive in it and it will be simpler for people to get started (unless we can think of a counter-case)